DependencyInjection/Configuration.php

```diff
@@ -157,6 +157,7 @@
  *   - [level]: level name or int value, defaults to DEBUG
  *   - [bubble]: bool, defaults to true
  *   - [ident]: string, defaults to
+ *   - [rfc]: string, defaults to
```

Comment:
defaults to ...? it does not have a default. also it's an integer

Reply:
It has, https://github.com/Seldaek/monolog/blob/master/src/Monolog/Handler/SyslogUdpHandler.php#L47

But yes, it is not correct to describe it as string